go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/gogo/protobuf v1.3.1
 	github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6
 	github.com/golang/mock v1.3.1 // indirect
+	github.com/golang/protobuf v1.3.2
```

Comment:
Don't need this, gogo/protobuf is a dropin replacement (three lines above).

Reply:
That was automatically added by `build.go proto`, should I remove it?

Reply:
Ah right, that's a "build tool dependency" which didn't yet get added because all those meddling with proto already had it installed previous to go modules :)  
I'd say yes, remove it - it's not needed for actually building, only development.

Reply:
Got it.

Reply:
Just noticed that the entry comes back with every build, so I'll leave it in for now.